resources/definitions/makeit_pro_l

```diff
@@ -0,0 +1,129 @@
+{
+    "id": "makeit_pro_l",
+    "version": 2,
+    "name": "MAKEiT_ProL",
```

Comment:
This is the user-visible string. You can use spaces and such in here, if needed.

The conventional spelling (and stylising) of this printer seems to be `MAKEiT Pro-L` or `Makeit Pro-L`.